tools/packaging/monkeyYaml.py

```diff
@@ -90,7 +90,7 @@ def myFlowList(value):
 def myMultiline(lines, value):
     # assume no explcit indentor (otherwise have to parse value)
     value = []
-    indent = 1
+    indent = -1
```

Comment:
It's legal for lines to be empty.

On Mon, Mar 30, 2015 at 3:01 PM, Erik Arvidsson <notifications@github.com>
wrote:

>> @@ -90,7 +90,7 @@ def myFlowList(value):
>>  def myMultiline(lines, value):
>>      # assume no explcit indentor (otherwise have to parse value)
>>      value = []
>> -    indent = 1
>> +    indent = -1
> Can this be?
> ```py
> indent = myLeadingSpaces(lines[0])
> ```
> or is there a case where lines is empty?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/tc39/test262/pull/216/files#r27431535